### PR TITLE
Setup minimal build of CRW 2.0

### PR DESCRIPTION
**Pull request by skabashnyuk**
Setup minimal build of CRW 2.0

### CODE REVIEW

File: assembly/codeready-workspaces-assembly-dashboard-war/src/main/webapp/assets/branding/DEPRECATED/branding.css [hunk 1]
```diff
@@ -1,6 +1,14 @@
 /**
  * Copyright (c) 2019 Red Hat, Inc.
-
+ * This program and the accompanying materials are made
```
Comment (nickboldt) [inline assembly/codeready-workspaces-assembly-dashboard-war/src/main/webapp/assets/branding/DEPRECATED/branding.css]: license now appears twice in the same file :(

File: assembly/codeready-workspaces-assembly-dashboard-war/src/main/webapp/assets/branding/CRW_wordmark-white.svg [hunk 2]
```diff
@@ -1,4 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
+<!--
```
Comment (nickboldt) [inline assembly/codeready-workspaces-assembly-dashboard-war/src/main/webapp/assets/branding/CRW_wordmark-white.svg]: I'm hoping these comments in SVG files won't impact performance

File: test/codeready-test-performance/pom.xml [hunk 3]
```diff
@@ -23,8 +23,8 @@
     <packaging>jar</packaging>
     <name>CRW :: Test :: Performance</name>
     <properties>
-        <jmeter.analysis.maven.plugin.version>1.0.6</jmeter.analysis.maven.plugin.version>
         <jmeter-maven-plugin.version>2.8.1</jmeter-maven-plugin.version>
+        <jmeter.analysis.maven.plugin.version>1.0.6</jmeter.analysis.maven.plugin.version>
```
Comment (nickboldt) [inline test/codeready-test-performance/pom.xml]: sort order changed?

File: assembly/codeready-workspaces-assembly-dashboard-war/src/main/webapp/assets/branding/DEPRECATED/branding.css (hunk 1 above)
Comment (skabashnyuk) [reply assembly/codeready-workspaces-assembly-dashboard-war/src/main/webapp/assets/branding/DEPRECATED/branding.css]: fixed

File: test/codeready-test-performance/pom.xml (hunk 3 above)
Comment (skabashnyuk) [reply test/codeready-test-performance/pom.xml]: yes. now it's sorted

File: assembly/codeready-workspaces-assembly-dashboard-war/src/main/webapp/assets/branding/DEPRECATED/redhat-logo.svg [hunk 4]
```diff
@@ -1,3 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (nickboldt) [inline assembly/codeready-workspaces-assembly-dashboard-war/src/main/webapp/assets/branding/DEPRECATED/redhat-logo.svg]: did you need to add this?

File: assembly/codeready-workspaces-assembly-dashboard-war/src/main/webapp/assets/branding/DEPRECATED/branding.css [hunk 5]
```diff
@@ -9,17 +9,6 @@
...
  */
- * All rights reserved. This program and the accompanying materials
- * are made available under the terms of the Eclipse Public License v2.0
- * which accompanies this distribution, and is available at
- * http://www.eclipse.org/legal/epl-2.0
- *
- * Contributors:
- *   Red Hat, Inc. - initial API and implementation
- */
-/**
```
Comment (nickboldt) [inline assembly/codeready-workspaces-assembly-dashboard-war/src/main/webapp/assets/branding/DEPRECATED/branding.css]: this file is deprecated -- could just delete it.

**Pull request merged**